setDefaultsPTB.m

```diff
@@ -0,0 +1,55 @@
+function cfg = setDefaultsPTB(cfg)
+
+% list the default values
+fieldsToSet.keyboard = [];
+fieldsToSet.responseBox = [];
+
+fieldsToSet.debug = true;
+fieldsToSet.testingTranspScreen = true;
+fieldsToSet.testingSmallScreen = true;
+
+fieldsToSet.backgroundColor = [0 0 0];
+
+fieldsToSet.textFont = 'Courier New';
+fieldsToSet.textSize = 18;
+fieldsToSet.textStyle = 1;
+
+fieldsToSet.monitorWidth = 42;
+fieldsToSet.screenDistance = 134;
+
+if isfield(cfg, 'initAudio') && cfg.initAudio
```

Comment:
@CerenB 
These would be the default values that would be chosen if the flag `cfg.initAudio` is true.

Do they make sense?

Reply:
yes it does. I do not know how/where you cal setDefault.m yet, but I've checked both initPTB and setDefault, they made sense. I'll let you know when I implement them in my scripts!

Reply:
It is called fairly early in initPTB (line 53)